### PR TITLE
Udp support

### PR DESCRIPTION
**Pull request by fsanaulla**
In this PR, was added UDP section in config generator.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -18,8 +18,8 @@ ext.devs = [
 apply plugin: 'java'
 apply plugin: 'jacoco'
 
-sourceCompatibility = 1.7
-targetCompatibility = 1.7
+sourceCompatibility = 1.8
```
Comment (aveuiller) [inline build.gradle]: Do you need features from java 1.8 ? I was trying to keep the compatibility with java 7

**Review by aveuiller — COMMENTED** [review]
Thank you for your contribution. If you don't use java 8 please set it back to 7, otherwise I can merge it. :)

**Comment by fsanaulla** [comment]
@aveuiller Done :)

**Comment by fsanaulla** [comment]
@aveuiller Can you release this changes? version  1.0.2?

**Pull request merged**